syntax = "proto3";

package test;

service TestService {
  rpc TestMethod (TestRequest) returns (TestResponse) {}
}

message TestRequest {
}
 

message TestResponse {
    bool status = 1;
}